Scripts/Editor/BaseWitWindow.cs

```diff
@@ -137,7 +137,7 @@ protected bool DrawWitConfigurationPopup()
 
             if (changed || witConfigs.Length > 0 && !witConfiguration)
             {
-                if (witConfigIndex < 0)
+                if (witConfigIndex < 0 || witConfigIndex >= witConfigs.Length)
```

Comment:
this was throwing some errors. I think it was a threading issue where the config index was updated before the updated list got to the baseWindow. This fixed it.